dags/pytorch_xla/pytorchxla2_torchbench.py

```diff
@@ -78,60 +78,56 @@
 
   # Running on V100 GPU
   config.get_torchbench_gpu_gke_config(
-      machine_type=resource.MachineVersion.N1_STANDARD_8,
+      machine_type=resource.MachineVersion.N1_STANDARD_16,
       image_family=resource.ImageFamily.COMMON_CU121_DEBIAN_11,
       accelerator_type=resource.GpuVersion.V100,
-      count=1,
-      use_xla2=True,
+      count=2,
```

Comment:
can you add a comment on why 2 GPUs are needed?

Reply:
Added, because the v100 GKE node is configured to use 2 GPUs. 2GPU's config will give us ~50GB CPU RAM.